packages/ui/src/apis/register.ts

```diff
@@ -19,9 +21,10 @@ export const register = async (username: string, password: string) => {
 
 export const sendRegisterSmsPasscode = async (phone: string) => {
   await api
-    .post(`${registerApiPrefix}/passwordless/sms/send-passcode`, {
+    .post(`${apiPrefix}/passwordless/sms/send`, {
       json: {
         phone,
+        flow: PasscodeType.Register,
```

Comment:
should we pass in the flow as an input param and merge all these apis?

Reply:
merged passwordless sign-in/register with email/sms routes.